libbeat/common/docker/helpers.go

```diff
@@ -0,0 +1,59 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package docker
+
+import (
+	"strings"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/common/safemapstr"
+)
+
+// ExtractContainerName strips the `/` characters that frequently appear in container names
+func ExtractContainerName(names []string) string {
+	output := names[0]
+
+	if len(names) > 1 {
+		for _, name := range names {
+			if strings.Count(output, "/") > strings.Count(name, "/") {
```

Comment:
Interesting that we take the name with less slashes :thinking:

Reply:
What's the chance of false-positives?

How about caching the count in a variable instead of computing it over and over again?